source/agg-pipeline-builder/create-a-view.txt

```diff
@@ -12,5 +12,44 @@ Create a View from Pipeline Results
    :depth: 1
    :class: singlecol
 
-TODO: 
+To quickly access the results of an aggregation pipeline without having to 
+execute an aggregation pipeline, you can create a :ref:`view <compass-views>` 
+on |compass|. Views are read-only, so they can help keep your data secure by 
+limiting user access to a predefined set of results. 
+
+About This Task 
+---------------
+
+Creating a view does not save the aggregation pipeline itself. 
+
+Procedure
```

Comment:
(info typing template)
I think we decided to use Steps rather than Procedure when you have stepped content.

Reply:
Sorry that's a habit I need to break, thanks for pointing it out!

Reply:
Also I went ahead and updated the other pages where the template was incorrect/not updated, so those changes will likely show up next round of review.

Reply:
No worries at all. Thank you for the updates. And sorry our template is in a sad state right now!